src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -429,19 +430,19 @@ public IEnumerable<Model3D> SceneItems
         /// <param name="parameters">A Watch3DViewModelStartupParams object.</param>
         /// <param name="logger">A logger to be used to log the exception.</param>
         /// <returns></returns>
-        public static DefaultWatch3DViewModel TryCreateHelixWatch3DViewModel(Watch3DViewModelStartupParams parameters, DynamoLogger logger)
+        public static DefaultWatch3DViewModel TryCreateHelixWatch3DViewModel(NodeModel model, Watch3DViewModelStartupParams parameters, DynamoLogger logger)
```

Comment:
please update the summary here to include the nodeModel parameter-  any reason you decided not to add this as an optional parameter that defaults to null?

Reply:
I will add the parameter summary here as I missed it. 

I did not use a default argument here (and in related places) for two main reasons;
1) There was already an existing pattern in the code base that was passing in the node as the first argument so I copied that pattern instead of creating two conflicting patterns.
2) Default arguments can be a cause for strange code behavior and for forcing arguments to be listed in a potentially unnatural way, due to the default arguments always being required to be at the end of the list of arguments. I generally do not like using default arguments as it can make the code less clear, and I have definitely run into strange issues using them in the past.